Lib/test/test_functools.py

```diff
@@ -635,7 +635,7 @@ def test_default_update(self):
         self.assertEqual(wrapper.__name__, 'f')
         self.assertEqual(wrapper.__qualname__, f.__qualname__)
         self.assertEqual(wrapper.attr, 'This is also a test')
-        self.assertEqual(wrapper.__annotations__['a'], 'This is a new annotation')
+        self.assertEqual(wrapper.__annotations__['a'], 'int')
```

Comment:
I'm surprised this had to be changed. I could imagine *adding* a test that it does this for `a: int`, but it should still work for `a: 'This is a new annotation'`?

Reply:
I guess I misinterpreted 'rewriting' as 'replacing' this test here https://github.com/python/cpython/pull/20434#discussion_r441918296. Will add a separate test

Reply:
Actually, do we need to add a separate test for this since they won't be much different? Either we fix this with `repr("This is a new annotation")` or we can just use `int`/`'int'` as a real type as you suggested before (and as it is presented here)

Reply:
Ah, gotcha. I had forgotten about my own comment (maybe it was one of the things I wrote weeks ago but didn't release until three days ago). Leaving this alone is fine.